storage/Devices/BcacheImpl.cc

```diff
@@ -53,17 +54,29 @@ namespace storage
 
     const char* DeviceTraits<Bcache>::classname = "Bcache";
 
+    const vector<string> EnumTraits<CacheMode>::names({
```

Comment:
Since this was new to you, would it help future contributors to add a doc comment to `Enum.h` how to use `EnumTraits`?

Reply:
well, I think it can help. But what would help the most is to know that such class exists at all.